packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -201,7 +206,7 @@ const props = defineProps({
 
 const { i18n } = composables.useI18n()
 const heightRef = ref<string>(props.height)
-const csvFilename = computed<string>(() => props?.chartTitle ? props.chartTitle : i18n.t('csvExport.defaultFilename'))
+const csvFilename = computed<string>(() => props?.chartTitle ?? props?.filenamePrefix ?? i18n.t('csvExport.defaultFilename'))
```

Comment:
Shouldn't it be `props.filenamePrefix ?? props.chartTitle || defaultFilename`?  It'd be weird if you set both `filenamePrefix` and `chartTitle` and the component ignored the filename prefix when determining the filename, and it'd be weird if an empty string chart title prevented the export from falling back to the default.

(I also don't think you need the question mark in `props?` -- that should always be defined, even if the exact prop is falsey.)

Reply:
Good point :)

Reply:
Updated the order.
 
```
props.filenamePrefix ?? (props.chartTitle || i18n.t('csvExport.defaultFilename'))
```